test/scripts/filters/external_link.js

```diff
@@ -26,13 +26,19 @@ describe('External link', () => {
       '# External link test',
       '1. External link',
       '<a href="https://hexo.io/">Hexo</a>',
-      '2. Internal link',
+      '2. External link with "rel" Attribute',
```

Comment:
suggest to add another test with `<a rel="noopener" href="https://hexo.io/">Hexo</a>` to be certain there would no duplicate `noopener`.

Reply:
OK, I will add that test cases.

Reply:
I might have missed line 32. Anyway, it's resolved.